project/SensibleSettings.scala

```diff
@@ -13,7 +13,8 @@ object Sensible {
 
   lazy val settings = Seq(
     ivyLoggingLevel := UpdateLogging.Quiet,
-    conflictManager := ConflictManager.strict,
+    // FIXME: how to be less strict for typesafe config? slick and akka pull in different versions
+    //conflictManager := ConflictManager.strict,
```

Comment:
you mean incompatible? Well there is nothing you can do when they are incompatible, we have to find a combination of all deps that actually work well together.